src/Http/Middleware/HandleErrorsWithWhoops.php

```diff
@@ -46,7 +46,8 @@ public function process(Request $request, Handler $handler): Response
                 $this->logger->error($e);
             }
 
-            return WhoopsRunner::handle($e, $request);
+            return WhoopsRunner::handle($e, $request)
+                ->withStatus($e->getCode() ?? 500);
```

Comment:
Can we make sure the code is in a valid range before we use it as a HTTP status? i.e. >= 400 && < 600